test/twitter/twits_test.exs

```diff
@@ -0,0 +1,99 @@
+defmodule Twitter.TwitsTest do
+  use TwitterWeb.ConnCase
+  # use Twitter.DataCase, async: true
```

Comment:
Is there a reason you aren't using datacase? 
Your Twits module is a context module and the way to test them is using DataCase: https://hexdocs.pm/phoenix/testing_contexts.html#the-datacase. It'll remove the `init_test_sessions`. 

Use ConnCase for controllers: https://hexdocs.pm/phoenix/Phoenix.ConnTest.html

Reply:
ok done